### DISCUSSION TITLE
How to add file attachement to model

### DISCUSSION THREAD

**Discussion by guiluxdj** [snipe/snipe-it#12494] [Q&A] [answered]
Hello,
I would like to add a file (ex: user manual) to a model but I don't see how. Is it possible ?

I see it's possible on a asset, but not on a model (more generic)

Thanks

**Comment by devvonmonahan** [comment, marked as answer]
This was added as a feature on [v6.0.6](https://github.com/snipe/snipe-it/releases/tag/v6.0.6)

What version are you currently on?

**Reply by guiluxdj** [reply]
I use v6.0.14 but i dont see this, option in model edit

**Reply by guiluxdj** [reply]
It's ok, it's not in the edit page but in the table. Great ;-)

**Reply by devvonmonahan** [reply]
This is where I go to upload files to a model; 
![image](https://user-images.githubusercontent.com/102929115/217767899-14b9cfdf-e2f3-4e22-8b25-afb694c920cd.png)
